### DISCUSSION TITLE
Filter & Sorting

### DISCUSSION THREAD

**Discussion by decryptedchaos** [dani-garcia/vaultwarden#4251] [Ideas]
I can go into more detail here if need be I'm not currently in a position to type a long winded description..  

Simply put,  i would like to see robust filtering in the web interface so that i might filter logins that are say older than _____     and many other scenarios,  such as filter by host,  etc etc     Also stacking filters would be useful  such as all logins older then ____ for host example.com

**Comment by BlackDex** [comment]
That is not something this project maintains.
It's part of the Bitwarden clients repo.

There already is a filtering feature available, see https://bitwarden.com/help/searching-vault/

**Comment by decryptedchaos** [comment]
> That is not something this project maintains. It's part of the Bitwarden clients repo.
> 
> There already is a filtering feature available, see https://bitwarden.com/help/searching-vault/

I appreciate the work being done here, and understand the huge issues that would arise from splitting away from upstream codebase too much..  

The issue (with Bitwarden too) is not being able to filter out old unused logins,   there's probably well over 10,000 logins in my database going back 15+ years its far too much work to go a tag all of them..  we need to be able to condense the database in a simple way

**Reply by BlackDex** [reply]
There is no last used timestamp for items, so that will be hard. And for that to work, all the client's need to support that feature.

There is a last update timestamp, but I'm not sure if you can filter or order on that.
